src/MorseL.Sockets/MorseLHttpMiddleware.cs

```diff
@@ -51,17 +51,14 @@ public class MorseLHttpMiddleware
     {
         private readonly ILogger _logger;
         private readonly RequestDelegate _next;
-        private readonly CancellationTokenSource _cts;
```

Comment:
If you really wanted to test this - you could add in an internal CTS factory that overrides the creation below so you can still cancel it.

Reply:
I do but it seemed a bit more churn than I wanted to introduce immediately prior to testing the other changes. I'll add it as a thing to do on #48